physionet-django/templates/event_home.html

```diff
@@ -115,6 +115,76 @@ <h5 class="modal-title">Participants</h5>
         </div>
     {% endfor %}
   {% endif %}
+
+<!-- Past events -->
+
+  <div class="card" >
+    <div class="card-header">
+      <h2>Past Events</h2>
+      <p class="card-text">Events hosted in the past</p>
```

Comment:
@ Yes, I agree. I added the subheader for the Past events because we had one for the Active events. But I also think we should remove them. I'll do it.